src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -149,12 +149,16 @@ private function registerAnnotationLoaders(ContainerBuilder $container)
     private function registerFileLoaders(ContainerBuilder $container)
     {
         $paths = [];
+        $globOptions = GLOB_BRACE | GLOB_NOSORT;
```

Comment:
Rather small peace of code, IMO it's too much to move this but if yall agree to, I'm willing to move it ;).